Automattic/woocommerce-payments
# Fix incorrect condition for domestic country check in checkout

Fixes #6971

#### Changes proposed in this Pull Request
Use domestic currency, instead of default deposit currency, to check if a payment method is compatible with the presentment currency in the checkout form. 









#### Testing instructions





* Create a CAN based merchant with USD as default currency (turn on multicurrency to add CAD currency as well)
* Enable Affirm and Afterpay payment methods.
* As a shopper,  add items to the cart (in USD) and proceed to checkout (more than USD 50 to get Affirm)
* Check that Affirm and Afterpay are not available in the checkout form.
* As a shopper, switch the currency to CAD.
* Check that Affirm and Afterpay are available after page reloads.
 
-------------------


**Post merge**
